pkg/apis/camel/v1alpha1/integrationplatform_types.go

```diff
@@ -105,6 +105,7 @@ type IntegrationPlatformBuildSpec struct {
 	PersistentVolumeClaim string                                  `json:"persistentVolumeClaim,omitempty"`
 	Maven                 MavenSpec                               `json:"maven,omitempty"`
 	HTTPProxySecret       string                                  `json:"httpProxySecret,omitempty"`
+	Cache                 bool                                    `json:"bool,omitempty"`
```

Comment:
One thing I noticed when testing - the `integrationplatform` spec appears like:

```
  spec:
    build:
      baseImage: fabric8/s2i-java:3.0-java8
      bool: true
```

You'll need to tweak the field name to `cache`.

Reply:
Thanks for the review, I have updated the code now